indica/equilibrium.py

```diff
@@ -77,8 +77,11 @@ def __init__(
 
         # Calculate rho-toroidal and 2D rho matrix
         self.rhot = np.sqrt(
-            (self.ftor - self.ftor.sel(rhop=0.0))
-            / (self.ftor.sel(rhop=1.0) - self.ftor.sel(rhop=0.0))
+            (self.ftor - self.ftor.sel(rhop=0.0, method="nearest"))
```

Comment:
Hadn't seen this during the previous iteration, there's something wrong and I think it's best to revert back to the main implementation.

The reason is that whatever equilibrium you read, it MUST have psin=0 and psin=1, hence it MUST have rhop=0 and rhop=1. 
If not, there must be something wrong with the equilibrium data...
Why did you have to implement this?